craft_parts/errors.py

```diff
@@ -49,6 +49,17 @@ def __str__(self) -> str:
         return "\n".join(components)
 
 
+class FeatureEnabled(PartsError):
```

Comment:
Can we consolidate this and `FeatureDisabled` into a single `FeatureError` or similarly named error class?

Reply:
Good idea.  I consolidated these two errors to `FeatureError`